docker/jenkins/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM php:7.2-fpm
```

Comment:
Minor comment:
Maybe it could be inside a jenkins/phpfpm folder.
What do you think?